packages/array/index.d.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Create a numeric range
+ *
+ * @param {Number} from
+ * @param {Number} to
+ * @return {Array}
+ *
+ * @example
+ * Array.range(-2, 2) // => [-2, -1, 0, 1, 2]
+ * Array.range(2, -2) // => [2, 1, 0, -1, -2]
+ */
+export declare function range(from: number, to: number): any[];
+/**
+ *
+ * Rotates a list a number of times. It"s completly agnostic about the
+ * contents of the list.
+ *
+ * @param {Integer} times - the number of rotations
+ * @param {Array} array
+ * @return {Array} the rotated array
+ * @example
+ * Array.rotate(1, [1, 2, 3]) // => [2, 3, 1]
+ */
+export declare function rotate(times: number, arr: any[]): any;
+/**
+ * Return a copy of the array with the null values removed
+ * @function
+ * @param {Array} array
+ * @return {Array}
+ *
+ * @example
+ * Array.compact(["a", "b", null, "c"]) // => ["a", "b", "c"]
+ */
+export declare const compact: (arr: any[]) => any[];
+/**
+ * Sort an array of notes in ascending order
+ *
+ * @param {String|Array} notes
+ * @return {Array} sorted array of notes
+ */
+export declare function sort(src: any[]): any;
```

Comment:
```suggestion
export declare function sort(src: string[]): string[];
```

With aliases should be something like this:
```ts
export declare function sort(src: string[]): note[];
```